libs/statistics/MastersRomanoWolfImproved.h

```diff
@@ -112,24 +113,27 @@ namespace mkc_timeseries
 	//                      is beaten by the max-of-all in that permutation.
 	// Bulk compute exceedance counts for every strategy once.
 
-	std::map<StrategyPtr, unsigned int> counts =
+	std::map<StrategyPtr, unsigned int> baseStatExceedanceCounts =
```

Comment:
nit: do we rely on the sorting nature of the `map` here?
Otherwise we can switch to `unordered_map`

Reply:
Technically we do, but we could use unordered_map, but I do not see need to switch. When I ran gprof a few weeks ago this was not an issue.